src/sass/modules/_m-additional-info.scss

```diff
@@ -2,3 +2,32 @@
   margin-top: 1em;
   margin-bottom: 1em;
 }
+
+.additional-info-button {
+  text-decoration: none;
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.additional-info-title {
+  color: $color-gray-dark;
+  border-bottom: 2px $color-primary dotted;
+  border-bottom-right-radius: 1px;
```

Comment:
This ensures that the ends/corners of the border are the same width as the rest of the border.